packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -677,6 +677,7 @@ export class WalletService {
    * @param {Object} opts
```

Comment:
https://github.com/bitpay/bitcore/pull/2395/files#diff-4741582acef9ff9bbe9ba1d9e4826fbaR535-R545

Change to:

`isMultiSig(coin)`

`isSingleAddress(coin)`